common/remote_util_test.cc

```diff
@@ -59,7 +59,8 @@ class RemoteUtilTest : public ::testing::Test {
 };
 
 TEST_F(RemoteUtilTest, BuildProcessStartInfoForSsh) {
-  ProcessStartInfo si = util_.BuildProcessStartInfoForSsh(kCommand);
+  ProcessStartInfo si =
+      util_.BuildProcessStartInfoForSsh(kCommand, ArchType::kLinux_x86_64);
   ExpectContains(si.command, {"ssh", kUserHostArg, kCommand});
```

Comment:
Should we also expect -t here?
Also maybe it make sense to add a test case for kWindows_x86_64

Reply:
Done.